datastore/pom.xml

```diff
@@ -57,10 +57,10 @@
   </dependencies>
   <build>
     <plugins>
-<!-- // [START maven]-->
+<!-- // [START datastore_maven]-->
```

Comment:
Seems pointless.

Reply:
At the team summit we are fixing all the region tags so they can be imported into the new system

Reply:
Ah, nope I meant the import of a blank line, not the change, I understand the change.